lib/types.ts

```diff
@@ -98,6 +98,10 @@ export interface EntrySys {
   contentType: Link
 }
 
+export type Metadata = {
+  tags?: Link[]
+} | undefined
```

Comment:
I think it's better to add `undefined` to the `getMetadata` and `onMetadataChanged` types below. That way we can properly use `Metadata` as a type when we know that it is set.

Reply:
Yeah, thought about the same and fixed it.